README.md

```diff
@@ -278,6 +286,9 @@ API. The commented interface is available in
 and
 [http](src/python/library/tritonclient/http/__init__.py).
 
+The Java client SDK provides similar capabilities as the Python SDK with similar classes and methods.
```

Comment:
Java client API ...

Given that it is Java isn't it more similar to the "C++ API". Even if not use "Python API" and not "SDK"